pkg/controllers/cluster/status.go

```diff
@@ -18,9 +18,8 @@ import (
 // It doesn't post the result to the API Server yet.
 // That will be done at the end of the sync loop.
 func (cc *ClusterReconciler) updateStatus(ctx context.Context, cluster *scyllav1alpha1.Cluster) error {
-	clusterStatus := scyllav1alpha1.ClusterStatus{
-		Racks: map[string]scyllav1alpha1.RackStatus{},
-	}
+	cluster.Status.Racks = map[string]scyllav1alpha1.RackStatus{}
```

Comment:
Unrelated change?

Reply:
It's related. Prior this PR ClusterStatus had single field, so rewriting it made sense. But now we added second field, and because of this code, it was every time zeroed out. I changed this function to modify only RackStatus and don't touch ManagerID which it isn't supposed to touch.